src/components/EventCard/EventCard.js

```diff
@@ -42,23 +42,26 @@ const EventCard = ({ event, className }) => {
 
   return (
     <li className={`${styles.card} ${className || ''}`}>
-      <Link to={`/event/${_id}`}>
-        <div className={styles.imageWrapper}>
+      <div className={styles.imageWrapper}>
+        <Link to={`/event/${_id}`}>
```

Comment:
That was to resolve https://github.com/ResistanceCalendar/resistance-calendar-frontend/issues/104

But I did just notice the <a> is the full width of that section of the card which is weird -- I'll include that in my next PR